filters/apiusagemonitoring/filter.go

```diff
@@ -37,6 +37,7 @@ const (
 type apiUsageMonitoringFilter struct {
 	Spec  *apiUsageMonitoringSpec
 	Paths []*pathInfo
+	UnknownPath *pathInfo // shouldn't this parameter(s) be privated?
```

Comment:
No comment please, better set it as a private member and later decide to have it public in case you need it for library users

Reply:
Is a question to the reviewer(s), whether I should change the visibility.